x-pack/plugins/watcher/public/lib/navigation.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
for my own knowledge... do we still need this now that we switched to react router?

Reply:
arguably not.  probably should be using history for nav.